src/stores/FrameStore.ts

```diff
@@ -759,11 +770,25 @@ export class FrameStore {
         this.animationChannelRange = [0, frameInfo.fileInfoExtended.depth - 1];
 
         if (this.isPVImage) {
-            this.astFrameSet = this.initPVImage();
+            this.astFrameSet = this.initFrame2D();
```

Comment:
we don't use `this.astFrameSet` anywhere outside of the constructor. I'd previously refactored this and turned it into a local variable, but I think it got lost in a merge commit or something :thinking: @duidae do you see any reason to have `this.astFrameSet` and `this.wcsInfo`?

Reply:
oh wait, ignore me... that's part of my PR  (#1362) :facepalm:

Reply:
Also refactored this part to shorten the code.